content/en/logs/processing/_index.md

```diff
@@ -121,6 +121,8 @@ Using the Datadog Agent or the RFC5424 format automatically sets the host value
 
 If a JSON formatted log file includes the `ddsource` attribute, Datadog interprets its value as the log's source. To use the same source names Datadog uses, see the [Integration Pipeline Reference][17].
 
+**Note**: Logs coming from a containerized environment require the use of an [environment variable][18] to override the default source and service values. 
```

Comment:
We should also add that those can be configured with the Autodiscovery labels: https://docs.datadoghq.com/agent/autodiscovery/integrations/?tab=kubernetes